packages/mobile/src/components/details-tile/DetailsTileNoAccess.tsx

```diff
@@ -30,12 +115,168 @@ export const DetailsTileNoAccess = ({
   trackId,
   premiumConditions
 }: DetailsTileNoAccessProps) => {
+  const styles = useStyles()
   const premiumTrackStatusMap = useSelector(getPremiumTrackStatusMap)
   const premiumTrackStatus = premiumTrackStatusMap[trackId] ?? null
 
-  if (premiumTrackStatus === 'UNLOCKING') {
-    return <DetailsTileUnlockingSection premiumConditions={premiumConditions} />
-  }
+  const { nftCollection, followee, tippedUser } =
+    useSpecialAccessEntity(premiumConditions)
+
+  const collectionLink = useMemo(() => {
+    if (!nftCollection) return ''
+
+    const { chain, address, externalLink } = nftCollection
+    if (chain === Chain.Eth && 'slug' in nftCollection!) {
+      return `https://opensea.io/collection/${nftCollection.slug}`
+    } else if (chain === Chain.Sol) {
+      const explorerUrl = `https://explorer.solana.com/address/${address}`
+      return externalLink ? new URL(externalLink).hostname : explorerUrl
+    }
+
+    return ''
+  }, [nftCollection])
+
+  const { onPress: handlePressCollection } = useLink(collectionLink)
+
+  const renderLockedDescription = useCallback(() => {
+    if (nftCollection) {
+      return (
+        <View style={styles.descriptionContainer}>
+          <Text style={styles.description}>
```

Comment:
I didn't go through this w a super fine toothed comb but it seems like there's a fair bit of of commonality between the rendered components here in each branch (i.e. `.descriptionContainer` + nested `.description`) and also in the `HasAccess` version of this component - is there any way to consolidate this at all?